docs/sdk/core-kit/mpc-core-kit/usage.mdx

```diff
@@ -51,6 +54,41 @@ Additional Helper Functions:
   - `mnemonicToKey()` - Converts a Mnemonic to a BN.
   - `keyToMnemonic()` - Converts a hex string to a Mnemonic.
 
+## Set TSS Wallet Index
+
+#### `setTssWalletIndex(accountIndex: number): void;`
+
+Sets the TSS Wallet Index for the User's account. This is a new feature that supports key derivation, allowing users to derive multiple private keys
```

Comment:
In mpc there are no Private keys, i think better description would be : "This function can be used to deterministically derive different wallet addresses by using account indexes. ". Using this function you can add multi address account support to your application.

Note: As an application, you need to persist this account index in application state. 

This feature is still in pre-release , so there can be some breaking changes in upcoming versions.

Reply:
Updated!